src/Pages/Signup/Signup.js

```diff
@@ -0,0 +1,414 @@
+import React, { Component } from 'react';
+import './Signup.scss';
+import Header from '../../Component/HeaderComponent/Header';
+import Footer from '../../Component/FooterComponent/Footer';
+import DaumPostCode from 'react-daum-postcode';
```

Comment:
- Import 순서 수정해주세요! 일반적인 convention을 따르는 이유도 있지만 순서만 잘 지켜주셔도 가독성이 좋아집니다. 아래 순서 참고해주세요.
- React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)
```suggestion
import React, { Component } from 'react';
import DaumPostCode from 'react-daum-postcode';
import Header from '../../Component/HeaderComponent/Header';
import Footer from '../../Component/FooterComponent/Footer';
import './Signup.scss';
```